README.md

```diff
@@ -142,6 +142,26 @@ Welcome to our source speech repository about [mobx-state-tree](https://github.c
 #### [#40](https://github.com/Dedetat/pimp-my-powerpuff/pull/40/files) - Reference an identifier
 
 ### Utils
+> Maybe there is some debugging tools we could use?
```

Comment:
We are sick of `console.log` everywhere, maybe **mobx-state-tree** can help us ? Yes it can! 😎
Let's see some debugging tools it provides

Reply:
(replace by this sentence)